src/stories/Foundation/Icon/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as icons from './icons';
+import { StandardColors } from '../../../shared/types';
+
+export interface IconProps {
+  className?: string;
+  color?: StandardColors;
+  size?: 24 | 48;
+  name: keyof typeof icons;
+}
+
+// name should accept the name of IconProps
+const getIcon = (name: IconProps['name'], size: IconProps['size']) => {
+  const formattedName = name.toLowerCase();
```

Comment:
Isn't the name already lowercase? At least the placeholder you made is lowercase?

Reply:
Yeah, it's mainly in case people put capitalized names as a prop, for redundancy. However it might be useless considering TypeScript. 
Will remove if you think it's not useful.

Reply:
typescript should take care of it 👍